waffle-mock-contract/README.md

```diff
@@ -75,7 +75,7 @@ You may wish to execute another contract through a mock.  Given the "AmIRichAlre
 ```js
 const contractFactory = new ContractFactory(AmIRichAlready.abi, AmIRichAlready.bytecode, sender);
 const amIRich = await contractFactory.deploy()
-const mockERC20 = await deployMockContract(sender, IERC20.abi);
```

Comment:
Why `ERC20` instead of `IERC20`? Wasn't the whole point no to implement the contract, just use interface?